lib/elixir/test/elixir/calendar_test.exs

```diff
@@ -260,6 +260,26 @@ defmodule DateTimeTest do
     assert DateTime.compare(datetime2, datetime1) == :gt
   end
 
+  test "convert/2" do
+    datetime_iso = %DateTime{year: 2000, month: 2, day: 29, zone_abbr: "CET",
+                             hour: 23, minute: 0, second: 7, microsecond: {0, 6},
```

Comment:
ideally this would be `{0, 0}` but when switching to and from Julian we end up with `{0, 6}`. By staying with `{0, 6}` as test inputs, the tests are more readable.